main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: leg
+//  action 1: the leg cramps
+leg.cramps();
+//  action 2: the leg moves
+leg.moves();
+//  action 3: the leg hurts strong
+leg.hurtsStrong();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: phone
+//  action 1: the phone rings
+phone.rings();
+//  action 2: the phone falls
+phone.falls();
+//  action 3: the phone drains so fast
+phone.drainsSoFast();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: tree
+//  action 1: the tree blooms
+tree.blooms();
+//  action 2: the tree grows
+tree.grows();
+//  action 3: the tree drys out 
+tree.drysOut();
+//  4) 
+//  Noun: horse
+//  action 1: the horse eats gras
+horse.eatsGras();
+//  action 2: the horse looses fur
+horse.loosesFur();
+//  action 3: the horse digests the gras
+horse.digestsTheGras();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: dog
+//  action 1: the dog barks
+dog.barks();
+//  action 2: the dog sleeps
+dog.sleeps();
+//  action 3: the dog catches the stick
+dog.catchesTheStick();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: waterTap
+//  action 1: the waterTap breaks
+waterTap.breaks();
+//  action 2: the waterTap leaks
+waterTap.leaks();
+//  action 3: the waterTap fills the sink
+waterTap.fillsTheSink();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: candle
+//  action 1: the candle melts
+candle.melts();
+//  action 2: the candle gets hot
+candle.getsHot();
+//  action 3: the candle illuminates room
+candle.illuminatesRoom();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.